source/_integrations/backup.markdown

```diff
@@ -72,15 +56,43 @@ automation:
 
 ## Restoring a backup
 
-<div class="note">
+Backups created via the **Backup** integration are located in your `/config/backups` directory.
+The {% term "Home Assistant Container" %} installation typically mounts this directory via `docker-compose.yml` or `docker run` to a directory of your choice.
 
-If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
+The steps on recovering from a backup depend on the installation type and use case. Follow one of these steps:
 
-</div>
+1. If you use {% term "Home Assistant Operating System" %} or {% term "Home Assistant Supervised" %}:
+   - The restore functionality is already built-in.
+   - Follow the steps described in [Restoring a backup](/common-tasks/os/#restoring-a-backup).
+2. If you use a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation:
+   - A Home Assistant backup is just a tar file of the `/config` directory, plus some metadata.
```

Comment:
```suggestion
   - A Home Assistant backup is just a tar file of the [configuration directory](/docs/configuration/#editing-configurationyaml), plus some metadata.
```
`/config` only exists in the containers, not for core so we should link to the explanation here.